### ISSUE TITLE
Convert to FAKE5

### ISSUE THREAD

**Issue by et1975** [elmish/elmish#148]
As a bonus would be great if the docs generation did not depend on net46

**Comment by zakaluka** [comment]
@et1975 Need some help if you have the time.  Facing an odd issue in AppVeyor, but I am not having any .NET SDK installation-related issues on my machine (I'm on Windows 7).  

Seems like every AppVeyor build is failing on installing the SDK.  I tried updating to 2.1.300 in the hopes that it would succeed.  I don't normally use AppVeyor and I'm having a hard time figuring out what is going on.

For the moment, I have reverted all references to SDK 2.1.4 (obviously still failing).

**Comment by zakaluka** [comment]
Ok, stopped and paid closer attention to the logs.  Appears that the real failure point is `restore`.  Trying some alternate syntax(es).

**Comment by zakaluka** [comment]
Well, this error is quite frustrating.

```
Fake.Core.BuildFailedException: Target 'Install' failed. ---> System.AggregateException: One or more errors occurred. ---> System.Exception: Start of process 'dotnet' failed. ---> System.ComponentModel.Win32Exception: ApplicationName='dotnet', CommandLine=' restore  Fable.Elmish.fsproj ', CurrentDirectory='/home/travis/build/elmish/elmish/src', Native error= Cannot find the specified file
```

Not sure why it can't find the `fsproj` file, unless `dotnet restore` doesn't want the file name when restoring (something specific to Linux, as this is working just fine on Windows?).

**Comment by zakaluka** [comment]
Ok, after 4 tries, I think that the system cannot find `dotnet` (should have realized this earlier, but I was sure it couldn't find the `fsproj`) :man_facepalming: 

Working on that issue now.

**Comment by zakaluka** [comment]
Ok, got to the point where I seem to be hitting fsprojects/Paket#3123 

Not sure if I can get AppVeyor to go past it.

**Comment by MangelMaxime** [comment]
@zakaluka  Thanks for giving this issue a try.

For `appveyor` problem, can you try to use something equivalent to [this one](https://github.com/MangelMaxime/Thoth/blob/master/appveyor.yml) ? 

I also added some comments in your current PR.

**Comment by zakaluka** [comment]
@MangelMaxime @et1975 

1. AppVeyor is successfully building now.  For now, it is running the `Build` target.

1. Travis is giving a pretty odd error message.  The specifics can be found in this [log file](https://api.travis-ci.org/v3/job/397623651/log.txt).  In short, the error states:

```
Could not find script './paket-files/Build/fsharp/FAKE/modules/Octokit/Octokit.fsx' in any paths searched. Searched paths:
["../../src/bin/Release/netstandard2.0"]
```

This error does not occur on AppVeyor, which seems to build just fine.  I tried a few variations on the `#load` statement in `build.fsx` in an attempt to navigate to the file, but none of them seemed to succeed.  Any idea what might be causing this?

**Comment by MangelMaxime** [comment]
@et1975 One note, I think we can't yet bootstrap FAKE using netcore runtime because we are using F#.Formatting which is running only for .Net I think.

**Comment by zakaluka** [comment]
Should this be closed since #149 was merged?
